src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,30 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include <cstring>
 
 
 using namespace std;
 
-int t07_ip() {
+int t07_ip(int argc, char* argv[]) {
+	char IP[16];
+	int result = 1;
+	cin >> IP;
 
+	char* nu = strtok(IP, ".");
+	while (nu != NULL)
+	{
+		if (atoi(nu) < 0 || atoi(nu) > 255)
+		{
+			result = 0;
+			break;
+		}
+		nu = strtok(NULL, ".");
+	}
+	if (result = 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'result' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4677716)